src/features/pilot_management/PilotSheet/components/LevelEditDialog.vue

```diff
@@ -115,15 +106,20 @@ export default Vue.extend({
 <style>
 .level-input {
   font-size: 55px;
-  width: 90px !important;
+  width: 120px !important;
   height: 65px !important;
+  line-height: 65px !important;
 }
 
-.level-input input {
+.level-input .v-select__selection {
   font-size: 55px;
   width: 90px !important;
   height: 65px !important;
-  max-height: 65px !important;
+  line-height: 65px !important;
+}
+
+.level-input input {
+  display: none;
```

Comment:
The `v-select` object had an "input" field that competed for space with the "selection" (though text input isn't available), causing the selection to overflow out of the `v-select` object.  Hiding this seemed to be the most reasonable solution, but if this will cause other issues (e.g. on mobile) then please let me know a preferred alternative.